lerobot/common/datasets/utils.py

```diff
@@ -20,7 +20,8 @@
 from itertools import accumulate
 from pathlib import Path
 from pprint import pformat
-from typing import Any
+from types import SimpleNamespace
+from typing import Any, Dict, Iterator
```

Comment:
Nit: since we're using python >=3.10 we don't need `typing.Dict` and we can type-hint directly with `dict`.
Similarly, typing.Iterator is [deprecated](https://docs.python.org/3/library/typing.html#typing.Iterable) and can be replaced by `collections.abc.Iterable`